wcc-contentful/lib/wcc/contentful/simple_client/management.rb

```diff
@@ -66,6 +66,34 @@ def webhook_definitions(**query)
     resp.assert_ok!
   end
 
+  def tags(query = {})
+    resp =
+      _instrument 'tags', query: query do
+        get("/spaces/#{space}/environments/#{environment}/tags", query)
+      end
+    resp.assert_ok!
+  end
+
+  def tag(key, query = {})
+    resp =
+      _instrument 'tags', tag: key, query: query do
+        get("/spaces/#{space}/environments/#{environment}/tags/#{key}", query)
+      end
+    resp.assert_ok!
+  end
+
+  def tag_create(tag_definition)
```

Comment:
This is just for future use, correct? I take it were just gonna create tags in the dashboard for now?

Reply:
There's a rake task in paper signs using this to update the list of "Ministry" tags